app/models/semester.rb

```diff
@@ -23,7 +23,11 @@ def can_be_destroyed?
     ! (student_applications.exists? || projects.exists?)
   end
 
+  def self.current_semester
+    where("is_current_semester = 'true'").first
+  end
+
   def self.clear_current_semester
-    Semester.where('is_current_semester').update_all("is_current_semester = 'false'")
+    where('is_current_semester').update_all("is_current_semester = 'false'")
```

Comment:
i saw it once on SO but i now realize is bad. and apparently vulnerable to sql injection